pandas/tests/io/test_packers.py

```diff
@@ -934,3 +934,7 @@ def test_msgpack_period_freq(self):
         s = Series(np.random.rand(5), index=date_range('20130101', periods=5))
         r = read_msgpack(s.to_msgpack())
         repr(r)
+
+    def test_msgpack_file_not_found(self):
+        bad_path = '1/2/3/4/path/does/not/exist'
+        pytest.raises(read_msgpack, FileNotFoundError, bad_path)
```

Comment:
use context manager or better still, i don't think you need to add a test.

 changing `TestCommonIOCapabilities.test_read_non_existant` and `TestCommonIOCapabilities.test_read_expands_user_home_dir` in `pandas/tests/io/test_common.py` would be sufficient.

Reply:
Ah, nice - looks like these tests handle this behavior already.